arbitrum-docs/welcome/get-started.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
Net-new doc to tie the new structure together for newcomers

Reply:
Oh, this is amazing! Great idea, @symbolpunk 💪